docs/using-gitbase/supported-languages.md

```diff
@@ -0,0 +1,517 @@
+- Puppet
+- PureBasic
+- PureScript
+- Python
+- Python console
+- QML
+- QMake
+- Quake
+- R
+- REALbasic
+- REXX
+- RPC
+- Racket
+- Ragel
+- Rascal
+- Reason
+- Rebol
+- Red
+- Redcode
+- Ren'Py
+- RenderScript
+- Ring
+- RobotFramework
+- Rouge
+- Ruby
+- Rust
+- SAS
+- SMT
+- SQF
+- SQLPL
+- Sage
+- SaltStack
+- Scala
+- Scheme
+- Scilab
+- Self
+- ShaderLab
+- Shell
+- ShellSession
+- Shen
+- Slash
+- Slice
+- Smali
+- Smalltalk
+- Smarty
+- Solidity
+- SourcePawn
+- Squirrel
+- Stan
+- Standard ML
+- Stata
+- SuperCollider
+- Swift
+- SystemVerilog
+- TI Program
+- TLA
+- TXL
+- Tcl
+- Tcsh
+- Terra
+- Thrift
+- Turing
+- TypeScript
+- Unified Parallel C
+- Unix Assembly
+- Uno
+- UnrealScript
+- UrWeb
+- VCL
+- VHDL
+- Vala
+- Verilog
+- Vim script
+- Visual Basic
+- Volt
+- WebAssembly
+- WebIDL
+- X10
+- XC
+- XProc
+- XQuery
+- XS
+- XSLT
+- Xojo
+- Xtend
+- YARA
+- Yacc
+- Zephir
+- Zimpl
+- eC
+- fish
+- mupad
+- nesC
+- ooc
+- q
+- sed
+- wdl
+- wisp
+- xBase
+
+### Markup
+- API Blueprint
+- Blade
+- CSS
+- Closure Templates
+- EJS
+- HTML
+- HTML+Django
+- HTML+ECR
+- HTML+EEX
+- HTML+ERB
+- HTML+PHP
+- HTML+Razor
+- Haml
+- Handlebars
+- Jupyter Notebook
+- Kit
+- Latte
+- Less
+- Liquid
+- MTML
+- Marko
+- Mask
+- Pic
+- PostCSS
+- PostScript
+- Pug
+- RAML
+- RHTML
+- RUNOFF
+- Roff
+- SCSS
+- SRecode Template
+- Sass
+- Scaml
+- Slim
+- Stylus
+- SugarSS
+- TeX
+- Tea
+- Twig
+- Vue
+- YASnippet
+
+### Data
+- ABNF
+- ASN.1
+- Adobe Font Metrics
+- Ant Build System
+- ApacheConf
+- C-ObjDump
+- COLLADA
+- CSON
+- CSV
+- Cloud Firestore Security Rules
+- CoNLL-U
+- Cpp-ObjDump
+- D-ObjDump
+- DNS Zone
+- Darcs Patch
+- Diff
+- EBNF
+- EML
+- Eagle
+- Easybuild
+- Ecere Projects
+- Edje Data Collection
+- FIGlet Font
+- Formatted
+- G-code
+- GN
+- Gerber Image
+- Glyph Bitmap Distribution Format
+- Gradle
+- Graph Modeling Language
+- GraphQL
+- Graphviz (DOT)
+- HAProxy
+- HTTP
+- HXML
+- INI
+- IRC log
+- JSON
+- JSON with Comments
+- JSON5
+- JSONLD
+- Java Properties
+- KiCad Layout
+- KiCad Legacy Layout
+- KiCad Schematic
+- Linker Script
+- Linux Kernel Module
+- Maven POM
+- NL
+- Nginx
+- Ninja
+- ObjDump
+- OpenType Feature File
+- Pickle
+- Protocol Buffer
+- Public Key
+- Pure Data
+- Python traceback
+- RPM Spec
+- Raw token data
+- Regular Expression
+- SPARQL
+- SQL
+- STON
+- SVG
+- Spline Font Database
+- SubRip Text
+- TOML
+- Turtle
+- Type Language
+- Unity3D Asset
+- Wavefront Material
+- Wavefront Object
+- Web Ontology Language
+- Windows Registry Entries
+- World of Warcraft Addon Data
+- X BitMap
+- X Font Directory Index
+- X PixMap
+- XCompose
+- XML
+- XPages
+- YAML
+- YANG
+- desktop
+- edn
+
+### Prose
+- AsciiDoc
+- Creole
+- Gettext Catalog
+- Markdown
+- MediaWiki
+- Org
+- Pod
+- Pod 6
+- RDoc
+- RMarkdown
+- Text
+- Textile
+- reStructuredText
+
+More details about aliases, groups, extensions, etc. you can find in [enry's repo](https://github.com/src-d/enry/blob/master/data/alias.go),
```

Comment:
Maybe all these explanations should come first, and the language list last? Because since it's very long, people may miss the distinction between enry-supported and bblfsh-supported.

Reply:
Maybe I'll leave just links and remove the list of all languages - WDYT?

Reply:
Ok, updated and rebased